tests/system/fim/common_tasks/delete_files.py

```diff
@@ -12,8 +12,25 @@
 import logging
 import os
 import time
-
-def delete_files(input_file_path, n, output_file_path, bunch_size=100, wait_time=100):
+def delete_file(path, attempt=0, sleep_time=5):
+    success = True
+    try:
+        if attempt > 0:
+          time.sleep(sleep_time)
+          logging.info(f"Failed to delete {path}, retry: {attempt}")
+        if attempt > 10:
+          success = False
```

Comment:
This should be `return False` directly to avoid the case when the file is actually deleted on attempt 11 and it still returns `False`.

Reply:
Ok, thanks! https://github.com/wazuh/wazuh-qa/pull/570/commits/e2936dcafee9410d734ae16cb532ceb72e621bcc